plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -955,4 +1057,24 @@ private boolean hasFabric8Dir() {
     private boolean isPomProject() {
         return "pom".equals(project.getPackaging());
     }
+
+    protected void writeResources(KubernetesList resources, ResourceClassifier classifier, Boolean generateRoute) throws MojoExecutionException {
+        // write kubernetes.yml / openshift.yml
+        File resourceFileBase = new File(this.targetDir, classifier.getValue());
+
+        File file = writeResourcesIndividualAndComposite(resources, resourceFileBase, this.resourceFileType, log, generateRoute);
+
+        // Attach it to the Maven reactor so that it will also get deployed
+        projectHelper.attachArtifact(project, this.resourceFileType.getArtifactType(), classifier.getValue(), file);
+
+        // TODO: Remove the following block when devops and other apps used by gofabric8 are migrated
```

Comment:
Not sure if this comment is still valid, since we don't have any plans to go further with gofabric8 and other stuff.

Reply:
not sure either, but I just copied over to consolidate some base classes. would like to do the clean up later when we remove any trace from gofabric8.

Reply:
okay